db/migrate/20150202215147_add_id_many_to_many.rb

```diff
@@ -0,0 +1,25 @@
+class AddIdManyToMany < ActiveRecord::Migration
+  def self.up
+    execute %{
+      ALTER TABLE "citations_sites" ADD "id" bigserial;
+      ALTER TABLE "citations_treatments" ADD "id" bigserial;
+      ALTER TABLE "inputs_runs" ADD "id" bigserial;
+      ALTER TABLE "inputs_variables" ADD "id" bigserial;
+      ALTER TABLE "managements_treatments" ADD "id" bigserial;
+      ALTER TABLE "pfts_priors" ADD "id" bigserial;
```

Comment:
@robkooper do these need to be bigserial? ESRI doesn't like it https://github.com/terraref/reference-data/issues/81 but there may be other options